src/azure-cli/azure/cli/command_modules/role/_help.py

```diff
@@ -508,6 +508,10 @@
 helps['ad sp create'] = """
 type: command
 short-summary: Create a service principal.
+long-summary: >-
+    **IMPORTANT**: The `az ad sp create` command can modify an existing application or service principal if
```

Comment:
Only `az ad app create` and `az ad sp create-for-rbac` can modify an existing application. `az ad sp create` will not.

Reply:
Changed